spec/tbd_tests_spec.rb

```diff
@@ -954,16 +954,18 @@
         next unless deratables.include?(id)
 
         # Evaluate current set content before processing a new linked surface.
-        is            = {}
-        is[:head    ] = set.keys.to_s.include?("head")
-        is[:sill    ] = set.keys.to_s.include?("sill")
-        is[:jamb    ] = set.keys.to_s.include?("jamb")
-        is[:corner  ] = set.keys.to_s.include?("corner")
-        is[:parapet ] = set.keys.to_s.include?("parapet")
-        is[:party   ] = set.keys.to_s.include?("party")
-        is[:grade   ] = set.keys.to_s.include?("grade")
-        is[:balcony ] = set.keys.to_s.include?("balcony")
-        is[:rimjoist] = set.keys.to_s.include?("rimjoist")
+        is               = {}
+        is[:head       ] = set.keys.to_s.include?("head")
+        is[:sill       ] = set.keys.to_s.include?("sill")
+        is[:jamb       ] = set.keys.to_s.include?("jamb")
+        is[:corner     ] = set.keys.to_s.include?("corner")
+        is[:parapet    ] = set.keys.to_s.include?("parapet")
+        is[:roof       ] = set.keys.to_s.include?("roof")
+        is[:party      ] = set.keys.to_s.include?("party")
+        is[:grade      ] = set.keys.to_s.include?("grade")
+        is[:balcony    ] = set.keys.to_s.include?("balcony")
+        is[:balconysill] = set.keys.to_s.include?("balconysill")
+        is[:rimjoist   ] = set.keys.to_s.include?("rimjoist")
```

Comment:
Few PR comments posted for this release, as the Test Suite is largely to support the related TBD PR. Most of the changes are brought ib to support ASHRAE 90.1 2022 (A10) requirements, summarized [here](https://github.com/rd2/tbd/pull/109/commits/4386168c7a208a1af34a537157ba0f31a84d10e4#r1335010855).